validator/src/main/java/com/amazon/aoc/validators/ValidatorFactory.java

```diff
@@ -47,6 +47,9 @@ public IValidator launchValidator(ValidationConfig validationConfig) throws Exce
         validator = new TraceValidator();
         expectedData = validationConfig.getExpectedTraceTemplate();
         break;
+      case "loadbalancing":
+        validator = new LoadBalanceValidator();
+        break;
```

Comment:
nit: If we are going to use `loadbalancing` here we should rename the validator to `LoadBalancingValidator`.

Reply:
Renamed to `LoadBalancingValidator`